scripts/setup-passcode.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+if [[ -z $OKTA_API_KEY ]]; then
+    echo "No Org API key"
+    exit 1
+fi
+
+if [[ -z $OKTA_DOMAIN ]]; then
+    echo "No Domain"
+    exit 1
+fi
+
+./scripts/oktamate users create -k $OKTA_API_KEY -d $OKTA_DOMAIN -g $PASSCODE_PASSWORD $PASSCODE_USERNAME
```

Comment:
You don't need to supply the key or domain on the command-line, as it will pick those up from the environment.  Also, you use `-p` or `--password` for the user's password, not `-g`

Reply:
> will pick those up from the environment

Not sure what do you mean.